aspnetcore/whats-new/dotnet-AspNetCore.Docs-mod5.md

```diff
@@ -1,82 +1,225 @@
+ms.custom: November-2023
+ms.date: 12/01/2023
 ---
 
-# ASP.NET Core docs: What's new for May 2023
+# ASP.NET Core docs: What's new for November 2023
 
-Welcome to what's new in the ASP.NET Core docs for May 2023. This article lists some of the major changes to docs during this period.
+Welcome to what's new in the ASP.NET Core docs for November 2023. This article lists some of the major changes to docs during this period.
 
 ## Blazor
 
 ### New articles
 
-- <xref:blazor/components/attribute-splatting>
-- <xref:blazor/components/key>
-- <xref:blazor/components/overwriting-parameters>
-- <xref:blazor/components/sync-context>
-- <xref:blazor/components/generic-type-support>
+- <xref:blazor/forms/binding>
+- <xref:blazor/forms/index>
+- <xref:blazor/components/class-libraries-with-static-ssr>
+- <xref:blazor/http-caching-issues>
+- [Title not found in: #31073](blazor/caching-issues.md)
+- <xref:blazor/security/server/static-server-side-rendering>
+- <xref:blazor/security/webassembly/standalone-with-identity>
+- <xref:blazor/forms/validation>
+- <xref:blazor/js-interop/ssr>
+- <xref:blazor/forms/input-components>
+- <xref:blazor/forms/troubleshoot>
 
 ### Updated articles
 
-- <xref:blazor/state-management> - State management article versioning
-- <xref:blazor/debug> - Blazor debug versioning update
-- <xref:blazor/globalization-localization> - Custom ICU loading
-- <xref:blazor/components/index>
-  - Component overview updates
-  - Components overview versioning
-- <xref:blazor/host-and-deploy/index> - Expand lead-in coverage on app base path
-- <xref:blazor/fundamentals/signalr> - Blazor SignalR fundamentals article versioning
-- <xref:blazor/blazor-ef-core> - Blazor Server EF Core article versioning
-- <xref:blazor/components/templated-components> - Blazor article versioning
-- <xref:blazor/tutorials/build-a-blazor-app> - Blazor article versioning
-- <xref:blazor/host-and-deploy/multiple-hosted-webassembly> - Subpath scenario for multiple hosted WASM apps
-- <xref:blazor/fundamentals/handle-errors> - Dispatch exceptions to the sync context
+- <xref:blazor/components/event-handling>
+  - Fix custom event example
+  - Update additional Blazor start examples
+  - Update event-handling.md
+- <xref:blazor/components/layouts> - Address layout namespaces
+- <xref:blazor/tutorials/build-a-blazor-app> - Fix example code
+- <xref:blazor/components/integration>
+  - Cross-link API
+  - Blazor :meat_on_bone: RTM 8.0
+  - Blazor language and note updates 8.0
+- <xref:blazor/components/js-spa-frameworks>
+  - Cross-link API
+  - Update additional Blazor start examples
+  - Blazor language and note updates 8.0
+- <xref:blazor/components/render-modes>
+  - Cross-link API
+  - Clarify global interactive WASM/Auto folders/files
+  - Environments + client services w/prerendering
+  - Blazor :meat_on_bone: RTM 8.0
+  - Blazor language and note updates 8.0
+  - JavaScript with Blazor static server rendering
+- <xref:blazor/components/sections> - Cross-link API
```

Comment:
@guardrex Can you make sense of this part of this section, from here down to line 118, in particular the ":meat_on_bone: RTM 8.0" text?